shimmer/learn/outputs.md

```diff
@@ -0,0 +1,115 @@
+---
+description: Stardust introduces changes to the UTXO model, which allow for new output types, features, and unlock conditions.
+image: /img/iota-wiki.png
+keywords:
+- explanation
+- shimmer
+- outputs
+- unlock conditions
+- features
+- output types
+
+---
+# Output Types and Features
+
+Stardust
+introduces [changes to the UTXO model](https://wiki.iota-community.org/shimmer/introduction/develop/explanations/what_is_stardust/rethink_utxo)
+that make it possible for outputs novel [output types](#output-types),  [unlock conditions](#unlock-conditions)
+and [features](#features).
+
+## Output Types
+
+### Basic Output
+
+The simplest output is the basic output. At the minimum, you will need to specify an address and amount of tokens.
+However, you can customize this basic output to include a variety
+of [features](#features) and [unlock conditions](#unlock-conditions).
+
+### NFT Output
+
+You can mint NFTs by creating NFT outputs and sending
+an [NFT transaction](https://wiki.iota-community.org/shimmer/introduction/develop/explanations/ledger/nft). You can
+define immutable metadata and a [verified issuer](#issuer) while you create it.
+
+### Alias Output
+
+An [alias](https://wiki.iota-community.org/shimmer/introduction/develop/explanations/ledger/alias) output is essentially
+a UTXO state machine that stores the state second layer smart contract chains and allows
+for two controllers, which can be rotated and create custom native tokens linked to the alias address.
+
+### Foundry Output
+
+You can create a foundry output to control the supply of
+your [native tokens](https://wiki.iota-community.org/shimmer/introduction/develop/explanations/ledger/foundry).
+
+## Unlock Conditions
+
+### Address
+
+The most traditional unlock condition is
+the [address](https://wiki.iota-community.org/shimmer/introduction/develop/explanations/what_is_stardust/unlock_conditions#address)
+.
+Addresses can only unlock outputs with their signatures.
+
+### Storage Deposit Return
+
+Due to the [storage deposit rules](welcome.md#storage-deposits), you cannot create transactions smaller than the minimum
+deposit. However, with the storage deposit return unlock condition, you can specify an amount that will need to be
+returned once the funds are unlocked. This way, you can send micro-transactions as small as you want. You will only need
+to send the minimum deposit, which will be returned once the micro-transaction is processed.
+
+### Expiration
+
+In Shimmer, you can define
+an [expiration date and time](https://wiki.iota-community.org/shimmer/introduction/develop/explanations/what_is_stardust/unlock_conditions#expiration)
+for an output. This way, if the output isn't consumed by the expiration
+date, it will be returned to your account. This is particularly useful if you want to cancel a transfer if a deadline
```

Comment:
To be precise. It would have never changed ownership. But I guess we can keep this as high level description